tests/integ/sagemaker/jumpstart/estimator/test_jumpstart_estimator.py

```diff
@@ -33,7 +33,7 @@
 from sagemaker.jumpstart.utils import get_jumpstart_content_bucket
 
 
-MAX_INIT_TIME_SECONDS = 5
+MAX_INIT_TIME_SECONDS = 15
```

Comment:
i'm curious, why was this needed?

Reply:
replied here: https://github.com/aws/sagemaker-python-sdk/pull/4467#discussion_r1509102642.  I think I'll go with approach 2 as it solves both this question and the assert_warning comment below